optics/src/Optics.hs

```diff
@@ -734,17 +734,25 @@ import Data.Either.Optics                    as P
 
 -- $otherresources
 --
--- * <https://skillsmatter.com/skillscasts/10692-through-a-glass-abstractly-lenses-and-the-power-of-abstraction Through a Glass, Abstractly: Lenses and the Power of Abstraction> a talk on the principles behind this library with <https://github.com/well-typed/optics/raw/master/Talk.pdf accompanying slides> by Adam Gundry (but note that the design details of @optics@ have changed substantially since this talk was given)
+-- Talks:
```

Comment:
should these be heading of some level, i.e.

```haskell
-- == Talks
```

https://hackage.haskell.org/package/haddock

Reply:
Done.